### DISCUSSION TITLE
Feature request: Add "Reload" command shortcut in System Tray's icon menu

### DISCUSSION THREAD

**Discussion by ghost** [input-leap/input-leap#1780] [Feature Requests]
> This issue has been migrated from old Barrier Github repository https://github.com/debauchee/barrier/issues/998
> 
> Issue created on: 2021-01-01 by @dev-101
> Issue last updated on: 2021-01-01

Hi, given the issue #997 I have, it would be awesome if tray icon has Reload command, as well. Thanks!

![Untitled](https://user-images.githubusercontent.com/5383059/103438444-78e13400-4c33-11eb-8a24-0e08e258ac15.png)

**Comment by bgbruno** [comment]
this icon menu item " reaload " really missing

in #macos I have to hit
1. show
2. show bottom " dock icons " 
3. click on app icon -- because this app window show under all windows all the times
4. stop
5. wait
6. start

really waste of my time circa 15 seconds many times of day
